### PR TITLE
[Pager] add offscreenLimit to PagerState Saver

### PR DESCRIPTION
**Pull request by wthee**
It looks like all properties need to be saved.

Fix: #545

### CODE REVIEW

File: pager/src/main/java/com/google/accompanist/pager/PagerState.kt [hunk 1]
```diff
@@ -666,14 +666,18 @@ class PagerState(
                 listOf<Any>(
                     it.pageCount,
                     it.currentPage,
+                    it.currentPageOffset,
```
Comment (chrisbanes) [inline pager/src/main/java/com/google/accompanist/pager/PagerState.kt:669]: We explicitly do not want the offset to be saved.

**Review by chrisbanes — CHANGES_REQUESTED** [review]

File: pager/src/main/java/com/google/accompanist/pager/PagerState.kt (hunk 1 above)
Comment (wthee) [reply pager/src/main/java/com/google/accompanist/pager/PagerState.kt:669]: Does `offscreenLimit `need to be saved?

Comment (chrisbanes) [reply pager/src/main/java/com/google/accompanist/pager/PagerState.kt:669]: I don't think so. The value is required to be set at init, and the value rarely changes.

Comment (wthee) [reply pager/src/main/java/com/google/accompanist/pager/PagerState.kt:669]: When navigating to a new page and then returning, the value of `PagerState.offscreenLimit` will always set to 1.  Just like the video of #545

Comment (chrisbanes) [reply pager/src/main/java/com/google/accompanist/pager/PagerState.kt:669]: Ah ok, let's keep that then!

**Review by chrisbanes — APPROVED** [review]

**Pull request merged**